tests/integration/bond_test.py

```diff
@@ -255,9 +255,26 @@ def test_set_bond_mac_address(eth1_up):
         bond99_cur_state = current_state[INTERFACES][0]
         assert bond99_cur_state[Interface.MAC] == MAC0.upper()
 
+        bond99_slaves_mac, eth1_slaves_mac, _ = return_bond_and_slaves_mac()
```

Comment:
IMHO it would be good if you implement `get_macs()`, so can use it like this here:

```
bond_mac, slave_mac = get_macs(statelib.show_only(('bond99', 'eth1')))
assert bond_mac == slave_mac == MAC0.upper()
```

Then you can also adjust the lines before this line.

Reply:
It is on a different branch and as spoken earlier, I'll keep this function and this commit and then create a PR from `get_all_macs branch` for all the previously submitted commits refactored with get_all_macs

Reply:
At line 254, we get the state from the system. You now need to get the slaves as well:
`current_state = statelib.show_only(('bond99', 'eth1', 'eth2'))`

Then you assert like we assert the bond mac (line 256).

Save below in the 2nd stage.